autofit/graphical/messages/abstract.py

```diff
@@ -310,27 +332,26 @@ def calc_exact_update(self, x: "AbstractMessage"):
 
     @classmethod
     def from_mode(cls, mode: np.ndarray, covariance: np.ndarray
-    ) -> "AbstractMessage":
+                  ) -> "AbstractMessage":
         pass
 
-    def log_normalisation(self, *dists: Union["AbstractMessage", float]
-    ) -> np.ndarray:
+    def log_normalisation(self, *elems: Union["AbstractMessage", float]
+                          ) -> np.ndarray:
         """
         Calculates the log of the integral of the product of a
         set of distributions
 
         NOTE: ignores log normalisation
         """
         # Remove floats from messages passed
-        dists = [
-            dist for dist in self._iter_dists(dists) 
+        dists: List[AbstractMessage] = [
+            dist for dist in self._iter_dists(elems)
             if isinstance(dist, AbstractMessage)]
 
         # Calculate log product of message normalisation
         log_norm = self.log_base_measure - self.log_partition
         log_norm += sum(
-            dist.log_base_measure - dist.log_partition
-            for dist in dists)
+            dist.log_base_measure - dist.log_partition for dist in dists)
```

Comment:
I'd put the closing parenthesis on the next line for visual balance. Just an OCD preference though ;)

Reply:
I think for multiline continuations it makes sense, but for one-liners it's nice to not let the code sprawl over too many lines. I like being able to read an entire function in one go.